NOTICE

```diff
@@ -0,0 +1,47 @@
+Fabric installer
+    Copyright 2021 LegacyFabric
```

Comment:
I would recommend updating the copyright notice similar to that of the recent Metadata changes. Ditto wherever this applies.